Modules/fcntlmodule.c

```diff
@@ -15,6 +15,33 @@
 #include <stropts.h>
 #endif
 
+#ifndef F_LINUX_SPECIFIC_BASE
+#    define F_LINUX_SPECIFIC_BASE      1024
+#endif
+
+/* fcntl.h */
+#ifndef F_ADD_SEALS
+
+/*
+ * Set/Get seals
+ */
+#define F_ADD_SEALS     (F_LINUX_SPECIFIC_BASE + 9)
```

Comment:
Rather of defining constants with fake values, it is better to add them conditionally, as other platform-specific constants. They should not be present on non-Linux and on old Linux.